<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>612</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="main_widget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QStackedWidget" name="page_stack">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="ConnectWidget" name="connect_page"/>
      <widget class="MainMenuWidget" name="main_menu"/>
      <widget class="OverviewWidget" name="overview"/>
      <widget class="TallyWidget" name="tally_page"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuNew">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="tearOffEnabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>&amp;New</string>
    </property>
    <addaction name="actionMember"/>
    <addaction name="actionSession"/>
   </widget>
   <addaction name="menuNew"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionMember">
   <property name="text">
    <string>&amp;Member</string>
   </property>
  </action>
  <action name="actionSession">
   <property name="text">
    <string>&amp;Session</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ConnectWidget</class>
   <extends>QWidget</extends>
   <header>ConnectWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MainMenuWidget</class>
   <extends>QWidget</extends>
   <header>MainMenuWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>OverviewWidget</class>
   <extends>QWidget</extends>
   <header>OverviewWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TallyWidget</class>
   <extends>QWidget</extends>
   <header>TallyWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
